excellent/functions/functions.go

```diff
@@ -1421,7 +1418,7 @@ func JSON(env utils.Environment, value types.XValue) types.XValue {
 //
 // @function format_date(date, [,format])
 func FormatDate(env utils.Environment, args ...types.XValue) types.XValue {
-	date, xerr := types.ToXDateTime(env, args[0])
+	date, xerr := types.ToXDate(env, args[0])
```

Comment:
this will currently get the date in the env timezone, which matches legacy behavior for things like `DAY()` etc